flink-runtime/src/test/java/org/apache/flink/runtime/jobmaster/slotpool/PhysicalSlotProviderImplTest.java

```diff
@@ -78,12 +76,11 @@ public void teardown() {
 	}
 
 	@Test
-	public void testBulkSlotAllocationFulfilledWithAvailableSlots()
-			throws InterruptedException, java.util.concurrent.ExecutionException, java.util.concurrent.TimeoutException {
+	public void testBulkSlotAllocationFulfilledWithAvailableSlots() throws InterruptedException, ExecutionException {
 		PhysicalSlotRequest request = createPhysicalSlotRequest();
 		addSlotToSlotPool();
 		CompletableFuture<PhysicalSlotRequest.Result> slotFuture = allocateSlot(request);
-		PhysicalSlotRequest.Result result = slotFuture.get(TIMEOUT.getSize(), TIMEOUT.getUnit());
+		PhysicalSlotRequest.Result result = slotFuture.get();
```

Comment:
What I had thought is to use `slotFuture.getNow(null)` and then doing a non-null check for the returned result.
Then we can say that allocation can be fulfilled immediately without delay if there are available slots.